File: Week_3/problem_9-Frequency_array.cpp
// https://codeforces.com/group/MWSDmqGsZm/contest/219774/problem/V

#include <iostream>
#include <vector>
using namespace std;

int main() {
    int n, m;
    cin >> n >> m;
    
    vector<int> ar(n);

    for (int i = 0; i < n; i++) {
        cin >> ar[i];
    }

    // Creating a vector of size m to hold the count index
    vector<int> count(m + 1, 0);

    // Increment count[value] if searched number matches
    for (int i = 0; i < n; i++) {
        int value = ar[i];
        count[value]++;
    }

    // Printing the count results
    for (int i = 1; i <= m; i++) {
        cout << count[i] << endl;
    }

    return 0;
}

File: Week_3/problem_9-Frequency_array.py
# https://codeforces.com/group/MWSDmqGsZm/contest/219774/problem/V

n, m = map(int, input().split())
ar = list(map(int, input().split()))

# Creating a list of size m+1 to hold the count index
count_arr = [0] * (m + 1)

# Increment count_arr[value] if searched number matches
for value in ar:
    count_arr[value] += 1

# Printing the count results
for i in range(1, m + 1):
    print(count_arr[i])
